src/modules/Elsa.Workflows.Core/VariableStorageDrivers/WorkflowInstanceStorageDriver.cs

```diff
@@ -17,8 +17,11 @@ public class WorkflowInstanceStorageDriver : IStorageDriver
     /// The key used to store the variables in the workflow state.
     /// </summary>
     public const string VariablesDictionaryStateKey = "Variables";
-
+    
+    /// <inheritdoc />
     public double Priority => 5;
+    /// <inheritdoc />
+    public StorageDriverTag[] Tags => [];
```

Comment:
Public list-type properties are types as `IEnumerable` or `ICollection` everywhere else, so it would be good to repeat that here. In this case, I would opt for an `IEnumerable`.